src/cli/bruker2dicom

```diff
@@ -58,13 +61,34 @@ def main():
     logging.getLogger().setLevel(verbosity.upper())
     
     function = arguments.pop("function")
+    source = arguments.pop("source")
+    
+    #create tmp directory in the current location
+    extractdir = tempfile.mkdtemp("","",".")
     try:
-        function(**arguments)
-    except Exception as e:
-        if verbosity == "debug":
-            raise
-        else:
-            parser.error(e)
+        # try to extract the archive
+        zip_obj = zipfile.ZipFile(source)
+        zip_obj.extractall(extractdir)
+    except:
+        # source is not a pvDataSet archive, so it should be a "normal" directory
+            arguments['source'] = source
```

Comment:
The indentation looks weird: is there a mix of tabs and spaces, or just extra spaces?